### DISCUSSION TITLE
Store different user's outputs seperatly

### DISCUSSION THREAD

**Discussion by Catmechanic457** [lllyasviel/Fooocus#3067] [Ideas]
Feature request: Ability to have separate log files for different users.

The issue is that after making some accounts using the `auth.json`, when clicking the `history` link, all generations by any user are shown.

**Comment by mashb1t** [comment]
You can disable the log with `--disable-image-log`.
A separation per user currently isn't implemented, but https://github.com/lllyasviel/Fooocus/issues/3041 exists, which may relate.

**Reply by Catmechanic457** [reply]
Thanks for that :)
Since the feature dosen't yet exist, I'll change the tag to "ideas"

**Reply by mashb1t** [reply]
@Catmechanic457 fyi a big question in general is if all enhancement "issues" should be migrated to discussions, category ideas... still not sure about that.

**Reply by Catmechanic457** [reply]
@mashb1t
Understood
Lmk if there's any edits I can make to my post that'll make things easier for you
